include/convex_bodies/hpolytope.h

```diff
@@ -1021,15 +1022,39 @@ class HPolytope {
             }
     }
 
-    template <typename update_parameters>
-    NT compute_reflection(Point &v, const Point &, DenseMT const &AE, VT const &AEA, NT const &vEv, update_parameters const &params) const {
+    // function to compute reflection for GaBW random walk
+    // compatible when the polytope is both dense or sparse
+    template <typename AE_type, typename update_parameters>
+    NT compute_reflection(Point &v, Point &p, AE_type const &AE, VT const &AEA, NT &vEv, update_parameters const &params) const {
```

Comment:
A comment about the signature of the function. I think it is good to separate the input from the output of the function. Here the output is `v, p, vEv` (is this true for `p` ?). Should we place all the input variables together (either in the start or end of the list of function parameters). Probably this will effect other functions as well so maybe it is too much work for this PR.

Reply:
I'll do it, I don't think this affects that many things, since I only use this function for GaBW